pick.lic

```diff
@@ -116,12 +116,16 @@ class LockPicker
         { name: 'pets', regex: /pets/, optional: true, description: 'Disarm boxes and place them in the pet box container.' },
         { name: 'count', regex: /\d+/, optional: true, description: 'How many pet boxes to make.' },
         { name: 'refill', regex: /refill/, optional: true, description: 'Refills your lockpick ring.' },
-        { name: 'source', regex: /(source=(\w+))/, optional: true, description: 'Container with boxes to pick.'}
+        { name: 'source', regex: /(source=(\w+))/, optional: true, description: 'Container with boxes to pick.'},
+        { name: 'force', regex: /force/i, optional: true, description: 'Force picking, ignoring any experience mindstate limits' },
```

Comment:
How does this arg differ from setting `stop_pick_on_mindlock` to false?

Reply:
This can be done on the fly, rather than having to change your YAML. I generally prefer to stop on mindlock, but if I'm e.g. opening boxes for someone else, that isn't the case.

Reply:
It is not different. If I set `stop_pick_on_mindlock: false` I keep picking when I hit 34/34.